### PR TITLE
Initial commit, missing test

### PR DESCRIPTION
**Pull request by mariarivera30**
*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

**Comment by szhangpitt** [comment]
> 12:16:55 Issue is that user id's for same people are different between qa and prod and the buggy code is grabbing the user id's from prod and the emails from qa. So it is grabbing the wrong emails from qa (because they are based on wrong user ids from prod)
> 12:18:09 So when INITIALLY syncing a project (meaning it does not exist yet in prod) we need to grab the ids and emails from QA (no other choice).
> 12:18:46 When we sync a project (that has already been synced in the past) we should grab the ids and emails from prod.

File: pybossa/syncer/project_syncer.py [hunk 1]
```diff
@@ -43,8 +43,10 @@ class ProjectSyncer(Syncer):
     def sync(self, project):
         target = self.get_target(short_name=project.short_name)
         payload = self._build_payload(project, target)
+        is_new_project = False
 
         if not target:
+            is_new_project = True
```
Comment (kbecker42) [inline pybossa/syncer/project_syncer.py]: Is this line of code used? It appears to return on the next line, without using it. The above comment would address this as well.

File: pybossa/syncer/project_syncer.py [hunk 2]
```diff
@@ -43,8 +43,10 @@ class ProjectSyncer(Syncer):
     def sync(self, project):
         target = self.get_target(short_name=project.short_name)
         payload = self._build_payload(project, target)
+        is_new_project = False
```
Comment (kbecker42) [inline pybossa/syncer/project_syncer.py]: The code is fine as-is. However, perhaps consider re-writing this line?

```python
is_new_project = not target
```

File: pybossa/syncer/project_syncer.py [hunk 3]
```diff
@@ -43,8 +43,10 @@ class ProjectSyncer(Syncer):
...
 
         if not target:
```
Comment (kbecker42) [inline pybossa/syncer/project_syncer.py]: If the above variable is changed per the comment, you could change this line to the positive case. :)

```python
if is_new_project:
  return self._create(payload, self.target_key)
else:
  ...
```

File: pybossa/syncer/project_syncer.py [hunk 4]
```diff
@@ -55,7 +57,7 @@ def sync(self, project):
         else:
             raise NotEnabled
 
-        return res
+        return is_new_project, res
```
Comment (kbecker42) [inline pybossa/syncer/project_syncer.py]: Just wanted to make sure this was intended. Is the function returning two different result types?

If the project is new, it returns: `return self._create(payload, self.target_key)`
If the project exists, it returns: `is_new_project, res`

Comment (mariarivera30) [reply pybossa/syncer/project_syncer.py]: That is fixed on the next PR that ill submit

**Pull request merged**